mocha.js

```diff
@@ -602,12 +615,17 @@ EventEmitter.prototype.emit = function (name) {
   }
 
   var handler = this.$events[name];
+  var args = [].slice.call(arguments, 1);
+
+  /* Practical Meteor changes*/
+  if (name !== 'any event')
```

Comment:
Document that this will also emit any event with whatever name (other than "any event") as an event with the name "any event"